.travis/run_task.py

```diff
@@ -54,6 +54,7 @@ def main():
         'miro_preprocessor-test',
         'lambdas-test',
         'monitoring_lambdas-test',
+        'sierra_adapter-test',
```

Comment:
Don't like this list, but not an issue for merge.

Reply:
Yeah, we should settle this publish/deploy disparity and then we can axe this.